tests/ExtractImportGraphTest.elm

```diff
@@ -32,33 +32,11 @@ baz = 1
 """
                 ]
                     |> Review.Test.runOnModules rule
-                    |> Review.Test.expectGlobalErrors
-                        [ importGraph
-                            [ ( [ "A" ]
-                              , [ [ "B" ], [ "C" ] ]
-                              )
-                            , ( [ "B" ]
-                              , [ [ "C" ] ]
-                              )
+                    |> Review.Test.expectDataExtract
+                        (Encode.object
+                            [ ( "onlineGraph", Encode.string "https://dreampuf.github.io/GraphvizOnline/#digraph%20%7B%0A%20%20%22A%22%20-%3E%20%7B%22B%22%20%22C%22%7D%0A%20%20%22B%22%20-%3E%20%7B%22C%22%7D%0A%7D" )
+                            , ( "graph", Encode.string "digraph {\n  \"A\" -> {\"B\" \"C\"}\n  \"B\" -> {\"C\"}\n}" )
                             ]
-                        ]
+                            |> Encode.encode 0
+                        )
         ]
-
```

Comment:
FWIW, I tend to use error functions like this so that the error message or the like can be changed in the future without a huge diff.  Less relevant for a pure JSON extract, probably.

Reply:
In tests I find that it makes things harder to read if you can't directly see the expected output. And also that it makes it harder to see when you've made a mistake in the source or tests files if it's behind a layer of logic.

But I agree about the big diff problem. That said, if commits are well-scoped (a commit only to change the message for instance), then a large diff is not a problem. That's my "by default" rule at least. Here since we only have a single test, I think creating a function with logic makes it harder to read.

(Not that this specific extract is easy to read though)

Reply:
(github seems to have lost my response, so apologies if it double posts)

Yeah, agreed, not trying to argue for its inclusion here, just explaining myself.